src/services/auth/change.ts

```diff
@@ -14,7 +14,7 @@ export default function change(req, res, next) {
     if (user == null) {
       return res.status(400).json({
         err: true,
-        msg: 'Token expired'
+        msg: 'Your login token has expired, please log in again'
```

Comment:
This endpoint is for password changing and token is from e-mail link, so I suggest something like `Your reset token has expired, please reset the password again`